### PR TITLE
feat: download/upload

### LINKED ISSUE #14: Import/Export project
Using a custom save file

### PR DESCRIPTION
**Pull request by belohlavek**
Kapture 2019-04-16 at 14 49 26

Closes #14

### CODE REVIEW

**Comment by belohlavek** [comment]
Fix overflow on project card tooltip

File: src/modules/project/types.ts [hunk 1]
```diff
@@ -13,3 +15,5 @@ export type Project = {
 export type Layout = { rows: number; cols: number }
 
 export type ProjectLayout = Layout & { hasError: boolean }
+
+export type SavedProject = { project: Project & { thumbnail?: string }; scene: Scene }
```
Comment (cazala) [inline src/modules/project/types.ts]: Should we name this `SaveFile`?

File: src/components/Modals/ImportModal/ImportModal.types.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+export type State = {
+  acceptedProjects: AcceptedProject[] | []
+  canImport: boolean
+}
+
+export type MapDispatchProps = Pick<Props, 'onImport'>
+
+export type AcceptedProject = {
```
Comment (cazala) [inline src/components/Modals/ImportModal/ImportModal.types.ts]: Should we name this `ImportedFile` ?

**Comment by twaldorf** [comment]
A few tiny copy changes from the feedback in #builder-testing, we are adding the .zip specifier back into the EN and ES copy in the drop-pad (it's in the description text in the CN and KR versions). Contained in Builder Localization.

**Pull request merged**